at_end2end_test/config/config.yaml

```diff
@@ -0,0 +1,9 @@
+# The @Protocol root server configuration
+root_server:
+  # The url to connect to root server
+  url: 'root.atsign.wtf'
+
+# cicd atSign details
+atSign:
+  firstAtSign: '@cicd1'
+  secondAtSign: '@cicd2'
```

Comment:
Remember that as soon as they are available we want to use different at signs than these ones which are also used for e2e test in at_server

Reply:
Sure Gary, Once we have the new atsign, will replace these with the new ones.